Modules/_io/iobase.c

```diff
@@ -79,21 +79,27 @@ iobase_unsupported(_PyIO_State *state, const char *message)
 
 /* Positioning */
 
-PyDoc_STRVAR(iobase_seek_doc,
-    "Change stream position.\n"
-    "\n"
-    "Change the stream position to the given byte offset. The offset is\n"
-    "interpreted relative to the position indicated by whence.  Values\n"
-    "for whence are:\n"
-    "\n"
-    "* 0 -- start of stream (the default); offset should be zero or positive\n"
-    "* 1 -- current stream position; offset may be negative\n"
-    "* 2 -- end of stream; offset is usually negative\n"
-    "\n"
-    "Return the new absolute position.");
+/*[clinic input]
```

Comment:
Usually there is an empty line before the list of arguments. It seems like it's not required. I'm not aware of coding style for argument clinic.

Reply:
Both styles are present in the code base, and there is no consistant style in Modules/_io. For sqlite3, I think all clinic code have the extra newline.

Reply:
My reference is the os module. I'm fine with having both coding styles until Argument Clinic becomes more strict or a coding style is defined (ex: in PEP 7).